diff --git a/.babelrc b/.babelrc
deleted file mode 100644
index c73df9d50b4998fc7b91c79f3a44eb7643196154..0000000000000000000000000000000000000000
diff --git a/.gitignore b/.gitignore
deleted file mode 100644
index 6a7d6d8ef6a9e8c5a09da8ab3584a96b189d5260..0000000000000000000000000000000000000000
diff --git a/.travis.yml b/.travis.yml
deleted file mode 100644
index 0968f7a4333372a7a58acf93cf096dad682c2a69..0000000000000000000000000000000000000000
diff --git a/VERSION b/VERSION
deleted file mode 100644
index 497b98347ef89fd72778a5b54cbb8ca3633c9467..0000000000000000000000000000000000000000
diff --git a/docs/AccessLevel.md b/docs/AccessLevel.md
deleted file mode 100644
index 980087c855905638e22be319769754dc43de9ea8..0000000000000000000000000000000000000000
diff --git a/docs/AccountStatusEnum.md b/docs/AccountStatusEnum.md
deleted file mode 100644
index 75333b8a137c468128e47f90b656670223acdc21..0000000000000000000000000000000000000000
diff --git a/docs/ApiKey.md b/docs/ApiKey.md
deleted file mode 100644
index b77d908654cf8349f8117c16222f61b781b13e66..0000000000000000000000000000000000000000
diff --git a/docs/ApiKeyPayload.md b/docs/ApiKeyPayload.md
deleted file mode 100644
index a6085ff347c8ce6b50c54ce3fcfd92aec91246ac..0000000000000000000000000000000000000000
diff --git a/docs/BodyContentType.md b/docs/BodyContentType.md
deleted file mode 100644
index c4a71659d1a3bc65b37332b14ed982630ed01d35..0000000000000000000000000000000000000000
diff --git a/docs/BodyPart.md b/docs/BodyPart.md
deleted file mode 100644
index a12f18fda438709db2509bddbbe22701321f2b35..0000000000000000000000000000000000000000
diff --git a/docs/Campaign.md b/docs/Campaign.md
deleted file mode 100644
index c9e78c74fdb0626f727f158df2f43686eb3b3e0d..0000000000000000000000000000000000000000
diff --git a/docs/CampaignOptions.md b/docs/CampaignOptions.md
deleted file mode 100644
index 9325335614f66973e091bc038f9d285048ecef4d..0000000000000000000000000000000000000000
diff --git a/docs/CampaignRecipient.md b/docs/CampaignRecipient.md
deleted file mode 100644
index ca772a41cbf5f034d42c0beb1bfecc327554e9db..0000000000000000000000000000000000000000
diff --git a/docs/CampaignStatus.md b/docs/CampaignStatus.md
deleted file mode 100644
index 305e4ea38e4ba72f0d938623de418198491f2df6..0000000000000000000000000000000000000000
diff --git a/docs/CampaignTemplate.md b/docs/CampaignTemplate.md
deleted file mode 100644
index 7f79d08c5f622a509d4e3010747369ccf6d71c06..0000000000000000000000000000000000000000
diff --git a/docs/CampaignsApi.md b/docs/CampaignsApi.md
deleted file mode 100644
index adffdac38b0660a511ccdacd6d9dcf8f43f335f5..0000000000000000000000000000000000000000
diff --git a/docs/ChannelLogStatusSummary.md b/docs/ChannelLogStatusSummary.md
deleted file mode 100644
index 806e09edd7e9bc090f8a85ed9147cfac7e5db8ea..0000000000000000000000000000000000000000
diff --git a/docs/CompressionFormat.md b/docs/CompressionFormat.md
deleted file mode 100644
index 075440257f8fdd9f14a3f06e3a3c6086acb6cddf..0000000000000000000000000000000000000000
diff --git a/docs/ConsentData.md b/docs/ConsentData.md
deleted file mode 100644
index 39c3d37caafc29d8bb66ee82b64d382b83335a6c..0000000000000000000000000000000000000000
diff --git a/docs/ConsentTracking.md b/docs/ConsentTracking.md
deleted file mode 100644
index 4e0f1fffc27b336db2f95e59d7c0c56eca55f1a9..0000000000000000000000000000000000000000
diff --git a/docs/Contact.md b/docs/Contact.md
deleted file mode 100644
index d37188b976ec3c4af412f62b11f0d1e4a5a2d943..0000000000000000000000000000000000000000
diff --git a/docs/ContactActivity.md b/docs/ContactActivity.md
deleted file mode 100644
index 3f46b8b7a4864bd85c572e36e861f32ee2f00301..0000000000000000000000000000000000000000
diff --git a/docs/ContactPayload.md b/docs/ContactPayload.md
deleted file mode 100644
index 382637deca80fc1353406616ef1624ca1a8b07e3..0000000000000000000000000000000000000000
diff --git a/docs/ContactSource.md b/docs/ContactSource.md
deleted file mode 100644
index 3368913eb9d43f67b367d5a3e83c2821a6cc2fd4..0000000000000000000000000000000000000000
diff --git a/docs/ContactStatus.md b/docs/ContactStatus.md
deleted file mode 100644
index 60e53a00e18d3ff11e0886cd44f13619ebc416dd..0000000000000000000000000000000000000000
diff --git a/docs/ContactUpdatePayload.md b/docs/ContactUpdatePayload.md
deleted file mode 100644
index b67155d221526a11591dde1bdc85954b6df935e1..0000000000000000000000000000000000000000
diff --git a/docs/ContactsApi.md b/docs/ContactsApi.md
deleted file mode 100644
index 6fd1cd4f0903caa666d4bbc71524be0016df3925..0000000000000000000000000000000000000000
diff --git a/docs/ContactsList.md b/docs/ContactsList.md
deleted file mode 100644
index 607c781a206aa6cc1a10643e1983ff8b95db478c..0000000000000000000000000000000000000000
diff --git a/docs/DeliveryOptimizationType.md b/docs/DeliveryOptimizationType.md
deleted file mode 100644
index 6d112db4f4d492be374a0210f357ebb45f747cb5..0000000000000000000000000000000000000000
diff --git a/docs/EmailContent.md b/docs/EmailContent.md
deleted file mode 100644
index efa73ea63ef5c02532d875117e3bb31613151a49..0000000000000000000000000000000000000000
diff --git a/docs/EmailData.md b/docs/EmailData.md
deleted file mode 100644
index 92488df54031cc965573286606477cc6622e954c..0000000000000000000000000000000000000000
diff --git a/docs/EmailJobFailedStatus.md b/docs/EmailJobFailedStatus.md
deleted file mode 100644
index 17929c56eaaad5f41b9ad263350afa83faabbcbd..0000000000000000000000000000000000000000
diff --git a/docs/EmailJobStatus.md b/docs/EmailJobStatus.md
deleted file mode 100644
index 9307e9ba4839216ad8f3b123a48c402ad16db394..0000000000000000000000000000000000000000
diff --git a/docs/EmailMessageData.md b/docs/EmailMessageData.md
deleted file mode 100644
index 64c57aeac8e75ce7bac2b72c57596d42416234eb..0000000000000000000000000000000000000000
diff --git a/docs/EmailPredictedValidationStatus.md b/docs/EmailPredictedValidationStatus.md
deleted file mode 100644
index 866f47ffa04c6a2209c66d48a7526c4138859670..0000000000000000000000000000000000000000
diff --git a/docs/EmailRecipient.md b/docs/EmailRecipient.md
deleted file mode 100644
index 6903b7145fb49c0a1788fb87d01b7e60d7030677..0000000000000000000000000000000000000000
diff --git a/docs/EmailSend.md b/docs/EmailSend.md
deleted file mode 100644
index 9331bf73c06ce90016290d81f8b689533b964268..0000000000000000000000000000000000000000
diff --git a/docs/EmailStatus.md b/docs/EmailStatus.md
deleted file mode 100644
index d45d4c179ae5d3f539dfb9e5abe9568c1d4dedc3..0000000000000000000000000000000000000000
diff --git a/docs/EmailTransactionalMessageData.md b/docs/EmailTransactionalMessageData.md
deleted file mode 100644
index 455880d6adce4458161060412ad1fb0be14ada6a..0000000000000000000000000000000000000000
diff --git a/docs/EmailValidationResult.md b/docs/EmailValidationResult.md
deleted file mode 100644
index b4044b4b9803962a58e3ea49e5d38379dd519128..0000000000000000000000000000000000000000
diff --git a/docs/EmailValidationStatus.md b/docs/EmailValidationStatus.md
deleted file mode 100644
index 5ae6e2f82c76c453dd107c704dbf1eba6ec80697..0000000000000000000000000000000000000000
diff --git a/docs/EmailView.md b/docs/EmailView.md
deleted file mode 100644
index 28849579bbd8118d535fd5cf3d2b6f5cb0000deb..0000000000000000000000000000000000000000
diff --git a/docs/EmailsApi.md b/docs/EmailsApi.md
deleted file mode 100644
index ae89cdfc2873ac6dab152879cb53f4f7b6d4f0f0..0000000000000000000000000000000000000000
diff --git a/docs/EmailsPayload.md b/docs/EmailsPayload.md
deleted file mode 100644
index d4c942105b502ae148516412eabd170ba8d4aaaf..0000000000000000000000000000000000000000
diff --git a/docs/EncodingType.md b/docs/EncodingType.md
deleted file mode 100644
index 350ccdf0f94ab8d27de54b690dd40c4fab4e8975..0000000000000000000000000000000000000000
diff --git a/docs/EventType.md b/docs/EventType.md
deleted file mode 100644
index e9617973508434c76eaa03e78a1d234f0f0c2fdf..0000000000000000000000000000000000000000
diff --git a/docs/EventsApi.md b/docs/EventsApi.md
deleted file mode 100644
index cc66b0963732629aba9a0628bd4818c68f909db6..0000000000000000000000000000000000000000
diff --git a/docs/EventsOrderBy.md b/docs/EventsOrderBy.md
deleted file mode 100644
index 684ac96cc409b4136567ef363d94be4dc1c8ebbb..0000000000000000000000000000000000000000
diff --git a/docs/ExportFileFormats.md b/docs/ExportFileFormats.md
deleted file mode 100644
index 16e785ce22cc78a704428af6291e624a2731d5bf..0000000000000000000000000000000000000000
diff --git a/docs/ExportLink.md b/docs/ExportLink.md
deleted file mode 100644
index adaf245bb42ff19a13e7927efd2555e838c9bd77..0000000000000000000000000000000000000000
diff --git a/docs/ExportStatus.md b/docs/ExportStatus.md
deleted file mode 100644
index f3fd4f115ecd020a36061a87f6746101b30644b1..0000000000000000000000000000000000000000
diff --git a/docs/FileInfo.md b/docs/FileInfo.md
deleted file mode 100644
index dcff3e807f01c14bb9e2fcb1d41c9338c3f5a7d0..0000000000000000000000000000000000000000
diff --git a/docs/FilePayload.md b/docs/FilePayload.md
deleted file mode 100644
index bc443d2414ded101c8708e5d6b37e969b4c72a09..0000000000000000000000000000000000000000
diff --git a/docs/FileUploadResult.md b/docs/FileUploadResult.md
deleted file mode 100644
index 95d238ff6cbacbe88dfaaff2672020a8e5078ff5..0000000000000000000000000000000000000000
diff --git a/docs/FilesApi.md b/docs/FilesApi.md
deleted file mode 100644
index d7141ceb3b1d977285ac2e7c689ddbe8ad50a01f..0000000000000000000000000000000000000000
diff --git a/docs/InboundPayload.md b/docs/InboundPayload.md
deleted file mode 100644
index 87629e4982418ee5980177242ba8c724894809a5..0000000000000000000000000000000000000000
diff --git a/docs/InboundRoute.md b/docs/InboundRoute.md
deleted file mode 100644
index 9ee6fa643444698b3aaa65b1edb1f6e2b14b449a..0000000000000000000000000000000000000000
diff --git a/docs/InboundRouteActionType.md b/docs/InboundRouteActionType.md
deleted file mode 100644
index 7fbb74546a4cb72521af478f5f232d62662f2542..0000000000000000000000000000000000000000
diff --git a/docs/InboundRouteApi.md b/docs/InboundRouteApi.md
deleted file mode 100644
index aa6fee5085f6c984d23a125bd43d209564bb4644..0000000000000000000000000000000000000000
diff --git a/docs/InboundRouteFilterType.md b/docs/InboundRouteFilterType.md
deleted file mode 100644
index 4d4fc7ae0a240b3fed6189b91c7287d35f54632c..0000000000000000000000000000000000000000
diff --git a/docs/ListPayload.md b/docs/ListPayload.md
deleted file mode 100644
index 805e78509ae4876cc8daa88f4a8c5d0b0264e375..0000000000000000000000000000000000000000
diff --git a/docs/ListUpdatePayload.md b/docs/ListUpdatePayload.md
deleted file mode 100644
index b23b42b02b9df1ef58633ce03316d0cdd41f0b45..0000000000000000000000000000000000000000
diff --git a/docs/ListsApi.md b/docs/ListsApi.md
deleted file mode 100644
index 4b287c4aa4612e089eba792ae58e395b7f93a871..0000000000000000000000000000000000000000
diff --git a/docs/LogJobStatus.md b/docs/LogJobStatus.md
deleted file mode 100644
index d4e20933b2a739cf980a58bb0daaaf9850ad3be2..0000000000000000000000000000000000000000
diff --git a/docs/LogStatusSummary.md b/docs/LogStatusSummary.md
deleted file mode 100644
index 366933325b7ce7136cc6c4bc2fc36198e2014b36..0000000000000000000000000000000000000000
diff --git a/docs/MergeEmailPayload.md b/docs/MergeEmailPayload.md
deleted file mode 100644
index c8310e5e6fc27e225e50d65a1caf1de295017f99..0000000000000000000000000000000000000000
diff --git a/docs/MessageAttachment.md b/docs/MessageAttachment.md
deleted file mode 100644
index 32dbbb6a3e08adae5c4b5d9388fd69f1e3ce581b..0000000000000000000000000000000000000000
diff --git a/docs/MessageCategory.md b/docs/MessageCategory.md
deleted file mode 100644
index 2bd5c1bc79abcede53e40e6719a1ad0a00f9e75b..0000000000000000000000000000000000000000
diff --git a/docs/NewApiKey.md b/docs/NewApiKey.md
deleted file mode 100644
index 94f2dfe4567901eb1d0540b3298adc1886cf5b16..0000000000000000000000000000000000000000
diff --git a/docs/NewSmtpCredentials.md b/docs/NewSmtpCredentials.md
deleted file mode 100644
index 7aa85df4630b509b20c605a892eacb21a3c5a7f5..0000000000000000000000000000000000000000
diff --git a/docs/Options.md b/docs/Options.md
deleted file mode 100644
index 9826f21d1f3c9a87e39692d0960e27d69a19c16b..0000000000000000000000000000000000000000
diff --git a/docs/RecipientEvent.md b/docs/RecipientEvent.md
deleted file mode 100644
index 8e2c8397d1b52113968de5e62ef93577da3686fb..0000000000000000000000000000000000000000
diff --git a/docs/SecurityApi.md b/docs/SecurityApi.md
deleted file mode 100644
index 396304c8efb7db80d59b6b2c1f5aeee475464b79..0000000000000000000000000000000000000000
diff --git a/docs/Segment.md b/docs/Segment.md
deleted file mode 100644
index b0964adbda6f32e57c9c8ee5e74b324b897beb4f..0000000000000000000000000000000000000000
diff --git a/docs/SegmentPayload.md b/docs/SegmentPayload.md
deleted file mode 100644
index 3b9328a8e79e4a63d55de796a9215ec2350f002e..0000000000000000000000000000000000000000
diff --git a/docs/SegmentsApi.md b/docs/SegmentsApi.md
deleted file mode 100644
index 341f4e9b228c0538fcb29b059f33ec6bdd549cba..0000000000000000000000000000000000000000
diff --git a/docs/SmtpCredentials.md b/docs/SmtpCredentials.md
deleted file mode 100644
index 17b3c9a6ad3a8e505afc6dd9c44688d37217834c..0000000000000000000000000000000000000000
diff --git a/docs/SmtpCredentialsPayload.md b/docs/SmtpCredentialsPayload.md
deleted file mode 100644
index 77802c53fdf7d7cb01066b0b8753b628fd34f30d..0000000000000000000000000000000000000000
diff --git a/docs/SortOrderItem.md b/docs/SortOrderItem.md
deleted file mode 100644
index 9afb57b124db9050df7381ed77f84bca0a54ef3b..0000000000000000000000000000000000000000
diff --git a/docs/SplitOptimizationType.md b/docs/SplitOptimizationType.md
deleted file mode 100644
index bdd62fdf63ec4c9ae25f8867ba31de465df6bfac..0000000000000000000000000000000000000000
diff --git a/docs/SplitOptions.md b/docs/SplitOptions.md
deleted file mode 100644
index c049813310876f168a80a33b61ba29b7387b4216..0000000000000000000000000000000000000000
diff --git a/docs/StatisticsApi.md b/docs/StatisticsApi.md
deleted file mode 100644
index bdeb8a1ef5e605ce7604e4647a13089fde3bb714..0000000000000000000000000000000000000000
diff --git a/docs/SubAccountInfo.md b/docs/SubAccountInfo.md
deleted file mode 100644
index ed8f48568379727b71e98178b6aa3ad46025e9fb..0000000000000000000000000000000000000000
diff --git a/docs/SubAccountsApi.md b/docs/SubAccountsApi.md
deleted file mode 100644
index 9d47e2e1c337983a113429b1181a2f5ed5669af3..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountEmailCreditsPayload.md b/docs/SubaccountEmailCreditsPayload.md
deleted file mode 100644
index b7312366e10237b7bffe5c8098657a7cd0304223..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountEmailSettings.md b/docs/SubaccountEmailSettings.md
deleted file mode 100644
index 7204adeab80f9cf2362599646807ed705ae5ffd3..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountEmailSettingsPayload.md b/docs/SubaccountEmailSettingsPayload.md
deleted file mode 100644
index 76e080f8706525b9a6aed87e0b078f63d64f8b02..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountPayload.md b/docs/SubaccountPayload.md
deleted file mode 100644
index 1b709906f6fe704e0c799383c7435934c6443a1d..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountSettingsInfo.md b/docs/SubaccountSettingsInfo.md
deleted file mode 100644
index 492027bdb41614ba0643ad1f41bbe6e8c8324ef7..0000000000000000000000000000000000000000
diff --git a/docs/SubaccountSettingsInfoPayload.md b/docs/SubaccountSettingsInfoPayload.md
deleted file mode 100644
index ca6d66e4da7e6d1a81ba16cbad1da20645010cef..0000000000000000000000000000000000000000
diff --git a/docs/Suppression.md b/docs/Suppression.md
deleted file mode 100644
index a849d04219bd94972744fe0a4930010518ee5b74..0000000000000000000000000000000000000000
diff --git a/docs/SuppressionsApi.md b/docs/SuppressionsApi.md
deleted file mode 100644
index a8f87ddd560d8e3d4b994050d6449bb4b52270d1..0000000000000000000000000000000000000000
diff --git a/docs/Template.md b/docs/Template.md
deleted file mode 100644
index 7c8555b921a2c7f9cbfd9819b233b6bb8ddf4d89..0000000000000000000000000000000000000000
diff --git a/docs/TemplatePayload.md b/docs/TemplatePayload.md
deleted file mode 100644
index 0de2611ec464295b620f2ab88d1b762b09bfc04b..0000000000000000000000000000000000000000
diff --git a/docs/TemplateScope.md b/docs/TemplateScope.md
deleted file mode 100644
index 93f88c992a2f09e762c1830224b8a9ef02ab9f37..0000000000000000000000000000000000000000
diff --git a/docs/TemplateType.md b/docs/TemplateType.md
deleted file mode 100644
index 08bd85f84c8c3c8f6c58664c37768cb9e4b48b26..0000000000000000000000000000000000000000
diff --git a/docs/TemplatesApi.md b/docs/TemplatesApi.md
deleted file mode 100644
index a2c49a5d8207182b764d85ac2c47023dc09429ad..0000000000000000000000000000000000000000
diff --git a/docs/TransactionalRecipient.md b/docs/TransactionalRecipient.md
deleted file mode 100644
index d3669e4c95f740b8d51c22142d46d6f40239732e..0000000000000000000000000000000000000000
diff --git a/docs/Utm.md b/docs/Utm.md
deleted file mode 100644
index 81b770d11c38a3d4e96ad71cf1cad7701e549427..0000000000000000000000000000000000000000
diff --git a/docs/VerificationFileResult.md b/docs/VerificationFileResult.md
deleted file mode 100644
index 2cec424aee811d9e1bfc801f1a7d7e3915fd66c3..0000000000000000000000000000000000000000
diff --git a/docs/VerificationFileResultDetails.md b/docs/VerificationFileResultDetails.md
deleted file mode 100644
index a846e6fbee4123a41be5d37258ebd49c37083bc5..0000000000000000000000000000000000000000
diff --git a/docs/VerificationStatus.md b/docs/VerificationStatus.md
deleted file mode 100644
index 4419899b9fc9ae5f6382ca5e074a3e799492dd71..0000000000000000000000000000000000000000
diff --git a/docs/VerificationsApi.md b/docs/VerificationsApi.md
deleted file mode 100644
index 406f28486241626e0e92bff02b79891dfb46bea8..0000000000000000000000000000000000000000
diff --git a/examples/README.md b/examples/README.md
deleted file mode 100644
index bba9334247903f074c249cf7181b828131c9109e..0000000000000000000000000000000000000000
diff --git a/examples/functions/addCampaign.js b/examples/functions/addCampaign.js
deleted file mode 100644
index b9c60751e2c56fa7b117f9ae91a32e66a897e33d..0000000000000000000000000000000000000000
diff --git a/examples/functions/addCampaign.md b/examples/functions/addCampaign.md
deleted file mode 100644
index d0e25515b0ea8fc2697e5af97f0a5dc4541ef02e..0000000000000000000000000000000000000000
diff --git a/examples/functions/addContacts.js b/examples/functions/addContacts.js
deleted file mode 100644
index 6fb31224b3fbe1469870b3b7e0347421d390153e..0000000000000000000000000000000000000000
diff --git a/examples/functions/addContacts.md b/examples/functions/addContacts.md
deleted file mode 100644
index cc875413989c8980733c9cd83823a1637e2f1792..0000000000000000000000000000000000000000
diff --git a/examples/functions/addList.js b/examples/functions/addList.js
deleted file mode 100644
index 40b44b02198a5c8bf064ea8bde4375a602c51b89..0000000000000000000000000000000000000000
diff --git a/examples/functions/addList.md b/examples/functions/addList.md
deleted file mode 100644
index 532d36ad2a17937fad9ca4b0f5d1e1a274561d4a..0000000000000000000000000000000000000000
diff --git a/examples/functions/addTemplate.js b/examples/functions/addTemplate.js
deleted file mode 100644
index 1af54ddb0645c38a40322297e683d63da157e535..0000000000000000000000000000000000000000
diff --git a/examples/functions/addTemplate.md b/examples/functions/addTemplate.md
deleted file mode 100644
index 45f6f158b2c93a9633ceeb9155e183fa0310494e..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteCampaign.js b/examples/functions/deleteCampaign.js
deleted file mode 100644
index d41650e4565caa29129d9601bb71ef8d506a2c75..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteCampaign.md b/examples/functions/deleteCampaign.md
deleted file mode 100644
index 634e4df40f795f495f9522bfc3c9cd4856bc49bc..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteContact.js b/examples/functions/deleteContact.js
deleted file mode 100644
index 3f4ae582d05696ece911932005e42eaf572381d0..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteContact.md b/examples/functions/deleteContact.md
deleted file mode 100644
index 71c63e2d2f6dbb9ca4f3ecec2a08a2edbabf5632..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteList.js b/examples/functions/deleteList.js
deleted file mode 100644
index 6619d881a46f277fe42747a9e69421bb8995274a..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteList.md b/examples/functions/deleteList.md
deleted file mode 100644
index a6e819206e8e0a469fb3bb210c4d0ef175fa5a6e..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteTemplate.js b/examples/functions/deleteTemplate.js
deleted file mode 100644
index 63b45b6a2164fb14bd336e1f70fe3881218add70..0000000000000000000000000000000000000000
diff --git a/examples/functions/deleteTemplate.md b/examples/functions/deleteTemplate.md
deleted file mode 100644
index 2d989aa3764a137932e34648d27676ebbeda30c5..0000000000000000000000000000000000000000
diff --git a/examples/functions/exportContacts.js b/examples/functions/exportContacts.js
deleted file mode 100644
index 6dca60a492dfb9982623e38556c1283f62deb3ba..0000000000000000000000000000000000000000
diff --git a/examples/functions/exportContacts.md b/examples/functions/exportContacts.md
deleted file mode 100644
index c2460c451ad29eac3daf9569e07be972fbccec04..0000000000000000000000000000000000000000
diff --git a/examples/functions/files/contacts.csv b/examples/functions/files/contacts.csv
deleted file mode 100644
index 992a4715f002d01496875e65839441f91a7faaaa..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadCampaign.js b/examples/functions/loadCampaign.js
deleted file mode 100644
index cc2fec55ce6b90ebf97b798e5f763e1caf327f9e..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadCampaign.md b/examples/functions/loadCampaign.md
deleted file mode 100644
index 2e97e28ce5815dff237b78ddb7428310ffa972ba..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadCampaignsStats.js b/examples/functions/loadCampaignsStats.js
deleted file mode 100644
index 8b5a1f2400ad9712dc0f5830c683517ab446ccc4..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadCampaignsStats.md b/examples/functions/loadCampaignsStats.md
deleted file mode 100644
index 277a9eab3c7d4983d191ffe8d066b76096620c85..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadChannelsStats.js b/examples/functions/loadChannelsStats.js
deleted file mode 100644
index dede74dd50e703af0ff55124a30d5e7ca2739b73..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadChannelsStats.md b/examples/functions/loadChannelsStats.md
deleted file mode 100644
index d75c8429f10bce674ab595f5d75fa5ff7f5c44f5..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadList.js b/examples/functions/loadList.js
deleted file mode 100644
index 67202cc3d899d50435d97f1cfb3277ae48493183..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadList.md b/examples/functions/loadList.md
deleted file mode 100644
index 828038db1362d3d9ca686fa4b3739fb66ed6b0a7..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadStatistics.js b/examples/functions/loadStatistics.js
deleted file mode 100644
index b8d8576364615de44ad58beb9985591a6d03bedf..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadStatistics.md b/examples/functions/loadStatistics.md
deleted file mode 100644
index b5977ee4d482230e4d6de1629717f90e4ef22b82..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadTemplate.js b/examples/functions/loadTemplate.js
deleted file mode 100644
index dc992fc3fb5ca6ffba3afcdbebd3694a2db5728d..0000000000000000000000000000000000000000
diff --git a/examples/functions/loadTemplate.md b/examples/functions/loadTemplate.md
deleted file mode 100644
index 7032273d5683dc6ecd7d0918c8e803ed22f50b3e..0000000000000000000000000000000000000000
diff --git a/examples/functions/sendBulkEmails.js b/examples/functions/sendBulkEmails.js
deleted file mode 100644
index 75751d6359de3fa65a5d0c1b2ea2a19f2a56c697..0000000000000000000000000000000000000000
diff --git a/examples/functions/sendBulkEmails.md b/examples/functions/sendBulkEmails.md
deleted file mode 100644
index ab4c94ddfe33371c81fe09aceb937729e36bf2e0..0000000000000000000000000000000000000000
diff --git a/examples/functions/sendTransactionalEmails.js b/examples/functions/sendTransactionalEmails.js
deleted file mode 100644
index 4cbdb2e32402d26c3b88d0fdc34c26668d4b9872..0000000000000000000000000000000000000000
diff --git a/examples/functions/sendTransactionalEmails.md b/examples/functions/sendTransactionalEmails.md
deleted file mode 100644
index eb519ef6b6b5b89836a934e41093cac0f280e46e..0000000000000000000000000000000000000000
diff --git a/examples/functions/updateCampaign.js b/examples/functions/updateCampaign.js
deleted file mode 100644
index b50000705146aa08b977bd3c5429197e29866838..0000000000000000000000000000000000000000
diff --git a/examples/functions/updateCampaign.md b/examples/functions/updateCampaign.md
deleted file mode 100644
index b476aca2c32474ccd2e0ad9ce17f18a77c4ea26f..0000000000000000000000000000000000000000
diff --git a/examples/functions/uploadContacts.js b/examples/functions/uploadContacts.js
deleted file mode 100644
index eca2a6a634351843d11c7ba731f2a44e9e47741c..0000000000000000000000000000000000000000
diff --git a/examples/functions/uploadContacts.md b/examples/functions/uploadContacts.md
deleted file mode 100644
index 7c67dea7aa45da348f9639df4180b8df1946a36c..0000000000000000000000000000000000000000
diff --git a/examples/package.json b/examples/package.json
deleted file mode 100644
index 2d70bd75bf131e25ad0affbdd55872e2a11ac635..0000000000000000000000000000000000000000
diff --git a/mocha.opts b/mocha.opts
deleted file mode 100644
index 907011807d680edb272de32bab832a628e2d5064..0000000000000000000000000000000000000000
diff --git a/package-lock.json b/package-lock.json
deleted file mode 100644
index 3a785eab4edecfa17be177381f8b328f601c33f7..0000000000000000000000000000000000000000
diff --git a/package.json b/package.json
index 619c4c5410ef7ae1c64720204f409eeae998a186..12592203dc083be50690de6756f27cab3ec61796 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@babel/cli": "^7.0.0",
-    "superagent": "^5.3.0"
+    "superagent": "^9.0.2"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
diff --git a/src/ApiClient.js b/src/ApiClient.js
deleted file mode 100644
index e171b398bcf0623ae6a90d135c9e309956b03f3e..0000000000000000000000000000000000000000
diff --git a/src/api/CampaignsApi.js b/src/api/CampaignsApi.js
deleted file mode 100644
index b981003423d94f14396735604604d6dd41ae3fe5..0000000000000000000000000000000000000000
diff --git a/src/api/ContactsApi.js b/src/api/ContactsApi.js
deleted file mode 100644
index 884bfc9c17505f089df20226dc06e0b7b153244d..0000000000000000000000000000000000000000
diff --git a/src/api/EmailsApi.js b/src/api/EmailsApi.js
deleted file mode 100644
index f854660b48c6aedcd63677aa4ae45f9930649644..0000000000000000000000000000000000000000
diff --git a/src/api/EventsApi.js b/src/api/EventsApi.js
deleted file mode 100644
index 2bacdddd379f55049d730455000fb2febbddbb1a..0000000000000000000000000000000000000000
diff --git a/src/api/FilesApi.js b/src/api/FilesApi.js
deleted file mode 100644
index ba5154ffc71146409c625466afe0a24f783f83a1..0000000000000000000000000000000000000000
diff --git a/src/api/InboundRouteApi.js b/src/api/InboundRouteApi.js
deleted file mode 100644
index d9b8edc3d61730c78bc0c61bd64384dffb594b0d..0000000000000000000000000000000000000000
diff --git a/src/api/ListsApi.js b/src/api/ListsApi.js
deleted file mode 100644
index 782bfec1e0a27c576a91ce2277ccfbe2d291ec7c..0000000000000000000000000000000000000000
diff --git a/src/api/SecurityApi.js b/src/api/SecurityApi.js
deleted file mode 100644
index 7ca02ed1894b2857e2c83954d15f45b502c97a4f..0000000000000000000000000000000000000000
diff --git a/src/api/SegmentsApi.js b/src/api/SegmentsApi.js
deleted file mode 100644
index aa1e4024c3c55f97dff8aed997b7776f787518c8..0000000000000000000000000000000000000000
diff --git a/src/api/StatisticsApi.js b/src/api/StatisticsApi.js
deleted file mode 100644
index 0dd65a6f9d44359a8701d1aeea0bc30beb7e7211..0000000000000000000000000000000000000000
diff --git a/src/api/SubAccountsApi.js b/src/api/SubAccountsApi.js
deleted file mode 100644
index c620bb741031fc25249fc449691e2f778de11608..0000000000000000000000000000000000000000
diff --git a/src/api/SuppressionsApi.js b/src/api/SuppressionsApi.js
deleted file mode 100644
index dbf50b08f74b1a7c30d8d86263124e682a2d0dec..0000000000000000000000000000000000000000
diff --git a/src/api/TemplatesApi.js b/src/api/TemplatesApi.js
deleted file mode 100644
index dde8781a64ead7a9ea73fe6a733ff7ed94ebf45d..0000000000000000000000000000000000000000
diff --git a/src/api/VerificationsApi.js b/src/api/VerificationsApi.js
deleted file mode 100644
index 3541375b392eb06ff403a53234bef53dd1fbdfcc..0000000000000000000000000000000000000000
diff --git a/src/index.js b/src/index.js
deleted file mode 100644
index 88030821324f01b59cbd2bf832e8f947f1491280..0000000000000000000000000000000000000000
diff --git a/src/model/AccessLevel.js b/src/model/AccessLevel.js
deleted file mode 100644
index a5da2ccceef7c1e21e131c033842bb4e15f0f8b5..0000000000000000000000000000000000000000
diff --git a/src/model/AccountStatusEnum.js b/src/model/AccountStatusEnum.js
deleted file mode 100644
index 681a7b4238b24060f40fbce218257edd027ae9f7..0000000000000000000000000000000000000000
diff --git a/src/model/ApiKey.js b/src/model/ApiKey.js
deleted file mode 100644
index d0a32f3f07cecf9f9df4717cbd5d39575bad0027..0000000000000000000000000000000000000000
diff --git a/src/model/ApiKeyPayload.js b/src/model/ApiKeyPayload.js
deleted file mode 100644
index efc37e4929501e643b2fbd328ea068a6f616233d..0000000000000000000000000000000000000000
diff --git a/src/model/BodyContentType.js b/src/model/BodyContentType.js
deleted file mode 100644
index 96948554671b93eb2733816e4a5f72a0a4c05415..0000000000000000000000000000000000000000
diff --git a/src/model/BodyPart.js b/src/model/BodyPart.js
deleted file mode 100644
index 5ace5505d08e23fb9fc77d653923c87e7e6c7bc7..0000000000000000000000000000000000000000
diff --git a/src/model/Campaign.js b/src/model/Campaign.js
deleted file mode 100644
index 94c3f2fc94d6f298b6ebbeaba77ea1ff2ecc35d4..0000000000000000000000000000000000000000
diff --git a/src/model/CampaignOptions.js b/src/model/CampaignOptions.js
deleted file mode 100644
index 9185aad97ebec008e34d9446b9d0684de6383206..0000000000000000000000000000000000000000
diff --git a/src/model/CampaignRecipient.js b/src/model/CampaignRecipient.js
deleted file mode 100644
index 2e8cd42d0059e9844bea6293a08583886ac31014..0000000000000000000000000000000000000000
diff --git a/src/model/CampaignStatus.js b/src/model/CampaignStatus.js
deleted file mode 100644
index 616ada09a7b533d17c726f636720f4f5ebba1c5d..0000000000000000000000000000000000000000
diff --git a/src/model/CampaignTemplate.js b/src/model/CampaignTemplate.js
deleted file mode 100644
index b5d2a0b3dbde4288b7a9f1b23bc5bcb51a52d2d0..0000000000000000000000000000000000000000
diff --git a/src/model/ChannelLogStatusSummary.js b/src/model/ChannelLogStatusSummary.js
deleted file mode 100644
index 4162ba24020e9217fbb7a46aa3471d10773c4eb0..0000000000000000000000000000000000000000
diff --git a/src/model/CompressionFormat.js b/src/model/CompressionFormat.js
deleted file mode 100644
index 6bcabb465bae3ab9b8db16f42c4e0cf9121a5505..0000000000000000000000000000000000000000
diff --git a/src/model/ConsentData.js b/src/model/ConsentData.js
deleted file mode 100644
index 05b8fc5de1791a0b7b8cfdeaf7901eb01124b828..0000000000000000000000000000000000000000
diff --git a/src/model/ConsentTracking.js b/src/model/ConsentTracking.js
deleted file mode 100644
index c64be820a9f4b504b8c6891b409352dc015957ef..0000000000000000000000000000000000000000
diff --git a/src/model/Contact.js b/src/model/Contact.js
deleted file mode 100644
index 0dee34bf50150d3675529c9b32be0d74b35c5dca..0000000000000000000000000000000000000000
diff --git a/src/model/ContactActivity.js b/src/model/ContactActivity.js
deleted file mode 100644
index e639ba6b84e8a86ed94adb0d00e6514cf4255e14..0000000000000000000000000000000000000000
diff --git a/src/model/ContactPayload.js b/src/model/ContactPayload.js
deleted file mode 100644
index 799e244eacfdf5df43eb29ac3e496585eaa8c044..0000000000000000000000000000000000000000
diff --git a/src/model/ContactSource.js b/src/model/ContactSource.js
deleted file mode 100644
index 20c132fe372d5199ea59142c6a3507649d910049..0000000000000000000000000000000000000000
diff --git a/src/model/ContactStatus.js b/src/model/ContactStatus.js
deleted file mode 100644
index 30c4246b1c9323eee169ac6336cefdfc36e80faa..0000000000000000000000000000000000000000
diff --git a/src/model/ContactUpdatePayload.js b/src/model/ContactUpdatePayload.js
deleted file mode 100644
index 95964f26367416e45605c2ac85a85a960567ab1a..0000000000000000000000000000000000000000
diff --git a/src/model/ContactsList.js b/src/model/ContactsList.js
deleted file mode 100644
index 995414273274bfa6bade192bdf1e5f3a036406d5..0000000000000000000000000000000000000000
diff --git a/src/model/DeliveryOptimizationType.js b/src/model/DeliveryOptimizationType.js
deleted file mode 100644
index 6fa19e51321daff02b6c35c3a93db1bfc7841c72..0000000000000000000000000000000000000000
diff --git a/src/model/EmailContent.js b/src/model/EmailContent.js
deleted file mode 100644
index 0e76c34da81a92f6d5ab74890f73f50fea06662c..0000000000000000000000000000000000000000
diff --git a/src/model/EmailData.js b/src/model/EmailData.js
deleted file mode 100644
index d2dddec5660661f83b06a3d895efcb5e977dd1c0..0000000000000000000000000000000000000000
diff --git a/src/model/EmailJobFailedStatus.js b/src/model/EmailJobFailedStatus.js
deleted file mode 100644
index 4f8ea75d0a7f45588b5eb6b9f07ced89ce2c1fad..0000000000000000000000000000000000000000
diff --git a/src/model/EmailJobStatus.js b/src/model/EmailJobStatus.js
deleted file mode 100644
index 93da232a240b117ac2ce54ff1f6bea0196c8c3f9..0000000000000000000000000000000000000000
diff --git a/src/model/EmailMessageData.js b/src/model/EmailMessageData.js
deleted file mode 100644
index 64982686132051ecc846c3e5650ecf1af50dce32..0000000000000000000000000000000000000000
diff --git a/src/model/EmailPredictedValidationStatus.js b/src/model/EmailPredictedValidationStatus.js
deleted file mode 100644
index 28d24813736a99ff2bdd7eb0461d21f29de80192..0000000000000000000000000000000000000000
diff --git a/src/model/EmailRecipient.js b/src/model/EmailRecipient.js
deleted file mode 100644
index 6be02da386e4ad8522e0b667ad55d4c14534e260..0000000000000000000000000000000000000000
diff --git a/src/model/EmailSend.js b/src/model/EmailSend.js
deleted file mode 100644
index c4e38c5e25f74c6fdd39b797b05da02ac34848eb..0000000000000000000000000000000000000000
diff --git a/src/model/EmailStatus.js b/src/model/EmailStatus.js
deleted file mode 100644
index 0b339898b962071b5d589e047f4d58b5af12c942..0000000000000000000000000000000000000000
diff --git a/src/model/EmailTransactionalMessageData.js b/src/model/EmailTransactionalMessageData.js
deleted file mode 100644
index 0b2cfcae93989340cf46618cb95299e0662b972e..0000000000000000000000000000000000000000
diff --git a/src/model/EmailValidationResult.js b/src/model/EmailValidationResult.js
deleted file mode 100644
index 05da7fe678167a7f8098e9c2f242cf2142eaf1cc..0000000000000000000000000000000000000000
diff --git a/src/model/EmailValidationStatus.js b/src/model/EmailValidationStatus.js
deleted file mode 100644
index 1887e351837721494b9124fbde4446362a75c1f0..0000000000000000000000000000000000000000
diff --git a/src/model/EmailView.js b/src/model/EmailView.js
deleted file mode 100644
index 6d6b06fb3470cbb24f56e78edb9a445188d99519..0000000000000000000000000000000000000000
diff --git a/src/model/EmailsPayload.js b/src/model/EmailsPayload.js
deleted file mode 100644
index 77f271e9a71e59661740ead7ea61438671da2f83..0000000000000000000000000000000000000000
diff --git a/src/model/EncodingType.js b/src/model/EncodingType.js
deleted file mode 100644
index 5b9f70320688c576be824acebcd326e26bce2ab4..0000000000000000000000000000000000000000
diff --git a/src/model/EventType.js b/src/model/EventType.js
deleted file mode 100644
index 5a57da2551fd03ac092255934ea99b5c6305e667..0000000000000000000000000000000000000000
diff --git a/src/model/EventsOrderBy.js b/src/model/EventsOrderBy.js
deleted file mode 100644
index 3d3d3a6807d8914163f05ca177e10c3a9f55f994..0000000000000000000000000000000000000000
diff --git a/src/model/ExportFileFormats.js b/src/model/ExportFileFormats.js
deleted file mode 100644
index 2e8040997414290a34ac1fc5e886519deaf58182..0000000000000000000000000000000000000000
diff --git a/src/model/ExportLink.js b/src/model/ExportLink.js
deleted file mode 100644
index cd5a76e5eb8b40ecfc80088e9303827f4a2856fc..0000000000000000000000000000000000000000
diff --git a/src/model/ExportStatus.js b/src/model/ExportStatus.js
deleted file mode 100644
index 5b95265b9412233104dac12fda623125da8ad693..0000000000000000000000000000000000000000
diff --git a/src/model/FileInfo.js b/src/model/FileInfo.js
deleted file mode 100644
index 68e57a6476ff873e9c40b5dc809b12168d9184ed..0000000000000000000000000000000000000000
diff --git a/src/model/FilePayload.js b/src/model/FilePayload.js
deleted file mode 100644
index f953cfd4d36a8f20a0a92032080b5fd641ce1f0a..0000000000000000000000000000000000000000
diff --git a/src/model/FileUploadResult.js b/src/model/FileUploadResult.js
deleted file mode 100644
index 647fe11f1420be7abd18b5f0d8079d4b7749307e..0000000000000000000000000000000000000000
diff --git a/src/model/InboundPayload.js b/src/model/InboundPayload.js
deleted file mode 100644
index dd15746f47a39cc81eadac6f0cfa406c848f3b43..0000000000000000000000000000000000000000
diff --git a/src/model/InboundRoute.js b/src/model/InboundRoute.js
deleted file mode 100644
index 9e556a85055f7aa992cbb43a693f867dc5024eb0..0000000000000000000000000000000000000000
diff --git a/src/model/InboundRouteActionType.js b/src/model/InboundRouteActionType.js
deleted file mode 100644
index 859a2dfbbac3ee8f53d1309023cf198bc27ab49b..0000000000000000000000000000000000000000
diff --git a/src/model/InboundRouteFilterType.js b/src/model/InboundRouteFilterType.js
deleted file mode 100644
index ed801b9ed584bd43d832be95db57899361181589..0000000000000000000000000000000000000000
diff --git a/src/model/ListPayload.js b/src/model/ListPayload.js
deleted file mode 100644
index 56beff4ec041fcb1b4812a5ba99c5ae5cff13c8d..0000000000000000000000000000000000000000
diff --git a/src/model/ListUpdatePayload.js b/src/model/ListUpdatePayload.js
deleted file mode 100644
index a9b2c29880ed07560e335179fafc77e564743ad0..0000000000000000000000000000000000000000
diff --git a/src/model/LogJobStatus.js b/src/model/LogJobStatus.js
deleted file mode 100644
index d6d4b7f5c7013f5b310a3275a2ffc2e73ec43445..0000000000000000000000000000000000000000
diff --git a/src/model/LogStatusSummary.js b/src/model/LogStatusSummary.js
deleted file mode 100644
index 7d37544abcc3d537856da8b4c12958fb841c0801..0000000000000000000000000000000000000000
diff --git a/src/model/MergeEmailPayload.js b/src/model/MergeEmailPayload.js
deleted file mode 100644
index 5756a63d7dc365d2f1b21af67a331615afcc5567..0000000000000000000000000000000000000000
diff --git a/src/model/MessageAttachment.js b/src/model/MessageAttachment.js
deleted file mode 100644
index 14a93177d5de2d033a043aeefae7dc1ec05cc3cd..0000000000000000000000000000000000000000
diff --git a/src/model/MessageCategory.js b/src/model/MessageCategory.js
deleted file mode 100644
index a2c3967ef0ba35c72d3ebf9cf6aed9c964275379..0000000000000000000000000000000000000000
diff --git a/src/model/NewApiKey.js b/src/model/NewApiKey.js
deleted file mode 100644
index 587534d3a3bb093765a84be334725c693b531967..0000000000000000000000000000000000000000
diff --git a/src/model/NewSmtpCredentials.js b/src/model/NewSmtpCredentials.js
deleted file mode 100644
index 4271a60da6e137a321c1fa96841b680e88bed77a..0000000000000000000000000000000000000000
diff --git a/src/model/Options.js b/src/model/Options.js
deleted file mode 100644
index d8d9b575018cbedff087d8b797574da2380a7f3e..0000000000000000000000000000000000000000
diff --git a/src/model/RecipientEvent.js b/src/model/RecipientEvent.js
deleted file mode 100644
index 33043bb58d5d5c812870ffd7304f22714900c1e0..0000000000000000000000000000000000000000
diff --git a/src/model/Segment.js b/src/model/Segment.js
deleted file mode 100644
index e4b91d8cfb1777265ed7fc161c493e32e2193be4..0000000000000000000000000000000000000000
diff --git a/src/model/SegmentPayload.js b/src/model/SegmentPayload.js
deleted file mode 100644
index 3a43903259877647105a45412db30943a7724844..0000000000000000000000000000000000000000
diff --git a/src/model/SmtpCredentials.js b/src/model/SmtpCredentials.js
deleted file mode 100644
index d8b1d25b229e99aa10c8c7ffb101f07ad2068001..0000000000000000000000000000000000000000
diff --git a/src/model/SmtpCredentialsPayload.js b/src/model/SmtpCredentialsPayload.js
deleted file mode 100644
index 3d2778b3e153621cad4f3a8c9514f28df538c291..0000000000000000000000000000000000000000
diff --git a/src/model/SortOrderItem.js b/src/model/SortOrderItem.js
deleted file mode 100644
index 6afc4b5ebe9b04b999b81d6936ba4c1e68c0b758..0000000000000000000000000000000000000000
diff --git a/src/model/SplitOptimizationType.js b/src/model/SplitOptimizationType.js
deleted file mode 100644
index 851a8b16f431b68f83a8afbe92e4d9326b2b2c92..0000000000000000000000000000000000000000
diff --git a/src/model/SplitOptions.js b/src/model/SplitOptions.js
deleted file mode 100644
index 3fecfe9d72bcf24e188ea621211da117169dbcbc..0000000000000000000000000000000000000000
diff --git a/src/model/SubAccountInfo.js b/src/model/SubAccountInfo.js
deleted file mode 100644
index a4f2f75b45b09cfb80de9dadc12343de9657d195..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountEmailCreditsPayload.js b/src/model/SubaccountEmailCreditsPayload.js
deleted file mode 100644
index f979f40493f0337f50316d14264fbd371681ffd0..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountEmailSettings.js b/src/model/SubaccountEmailSettings.js
deleted file mode 100644
index 60207079178695bdecfe05a7904cb08245425fc6..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountEmailSettingsPayload.js b/src/model/SubaccountEmailSettingsPayload.js
deleted file mode 100644
index 5955b40137a0c490e9bdbfeb81a6aa76ff4b26ea..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountPayload.js b/src/model/SubaccountPayload.js
deleted file mode 100644
index ce2f550947b85124d903c9909e15097b24218230..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountSettingsInfo.js b/src/model/SubaccountSettingsInfo.js
deleted file mode 100644
index 4da9dbaf8d7393022bd451a4f576111c31f67b3c..0000000000000000000000000000000000000000
diff --git a/src/model/SubaccountSettingsInfoPayload.js b/src/model/SubaccountSettingsInfoPayload.js
deleted file mode 100644
index 00963133644d5b169a85df6c88419ba4eeab2278..0000000000000000000000000000000000000000
diff --git a/src/model/Suppression.js b/src/model/Suppression.js
deleted file mode 100644
index fb91e298da08949fbcf0b39cce84ed14e89b0c7d..0000000000000000000000000000000000000000
diff --git a/src/model/Template.js b/src/model/Template.js
deleted file mode 100644
index eaeeeb445c8bab10e51c2a588e3ffc7183e5cc0f..0000000000000000000000000000000000000000
diff --git a/src/model/TemplatePayload.js b/src/model/TemplatePayload.js
deleted file mode 100644
index b932c1215a09dfbd0509c4f6e06d302449293880..0000000000000000000000000000000000000000
diff --git a/src/model/TemplateScope.js b/src/model/TemplateScope.js
deleted file mode 100644
index 4796292d6fa7d7b51fc29039f90738e86f1d80c1..0000000000000000000000000000000000000000
diff --git a/src/model/TemplateType.js b/src/model/TemplateType.js
deleted file mode 100644
index 7c3eb346412476f04a9eaca275bd2848a6ff08d2..0000000000000000000000000000000000000000
diff --git a/src/model/TransactionalRecipient.js b/src/model/TransactionalRecipient.js
deleted file mode 100644
index c87a666d4735d97c6d201fbd0f8434e1e21ff274..0000000000000000000000000000000000000000
diff --git a/src/model/Utm.js b/src/model/Utm.js
deleted file mode 100644
index d0ff469102d451254941aef5dbca672515d671ae..0000000000000000000000000000000000000000
diff --git a/src/model/VerificationFileResult.js b/src/model/VerificationFileResult.js
deleted file mode 100644
index 705b5fc28f87c3dc6d5a2cd906c242a1ac606c75..0000000000000000000000000000000000000000
diff --git a/src/model/VerificationFileResultDetails.js b/src/model/VerificationFileResultDetails.js
deleted file mode 100644
index ad258666d1e071cbaa28718c575efd78d782b3c2..0000000000000000000000000000000000000000
diff --git a/src/model/VerificationStatus.js b/src/model/VerificationStatus.js
deleted file mode 100644
index ddca74f46f031fd78d38f98ee60ea82e35a3fc7d..0000000000000000000000000000000000000000
diff --git a/test/api/CampaignsApi.spec.js b/test/api/CampaignsApi.spec.js
deleted file mode 100644
index 73b59b54d3a0cd9f1a8b5ad3503a4390c7b0bd07..0000000000000000000000000000000000000000
diff --git a/test/api/ContactsApi.spec.js b/test/api/ContactsApi.spec.js
deleted file mode 100644
index 31b67522ad9ed89c79cd22fd8e51ee6a93f0673a..0000000000000000000000000000000000000000
diff --git a/test/api/EmailsApi.spec.js b/test/api/EmailsApi.spec.js
deleted file mode 100644
index 0712e5cefeae51a76453759ff6b2c115b7d72973..0000000000000000000000000000000000000000
diff --git a/test/api/EventsApi.spec.js b/test/api/EventsApi.spec.js
deleted file mode 100644
index 71afb460d10ac7439e2121f36965f684b41d1f79..0000000000000000000000000000000000000000
diff --git a/test/api/FilesApi.spec.js b/test/api/FilesApi.spec.js
deleted file mode 100644
index 80733807fa2fcab6874871b59aa998bf51e07284..0000000000000000000000000000000000000000
diff --git a/test/api/InboundRouteApi.spec.js b/test/api/InboundRouteApi.spec.js
deleted file mode 100644
index 9ebdd3d543b83f6ca5da3c3516dc403aba845bb0..0000000000000000000000000000000000000000
diff --git a/test/api/ListsApi.spec.js b/test/api/ListsApi.spec.js
deleted file mode 100644
index c671ffc0540d28288b1978be92f9d64b4c05bfa4..0000000000000000000000000000000000000000
diff --git a/test/api/SecurityApi.spec.js b/test/api/SecurityApi.spec.js
deleted file mode 100644
index bd661eb04c7b78c71713bb2c52ef5c5d12d34c58..0000000000000000000000000000000000000000
diff --git a/test/api/SegmentsApi.spec.js b/test/api/SegmentsApi.spec.js
deleted file mode 100644
index 692a35029e77688558cf776922e753fae6d415b0..0000000000000000000000000000000000000000
diff --git a/test/api/StatisticsApi.spec.js b/test/api/StatisticsApi.spec.js
deleted file mode 100644
index 7c6f1d12e24ddfb897eeeb6b5e8d2bcd868df735..0000000000000000000000000000000000000000
diff --git a/test/api/SubAccountsApi.spec.js b/test/api/SubAccountsApi.spec.js
deleted file mode 100644
index d0dc2b8a86d0633b37e13de84ba7bd64766604af..0000000000000000000000000000000000000000
diff --git a/test/api/SuppressionsApi.spec.js b/test/api/SuppressionsApi.spec.js
deleted file mode 100644
index 5aaf801149e422e782e3a2812104c1147c27a1ad..0000000000000000000000000000000000000000
diff --git a/test/api/TemplatesApi.spec.js b/test/api/TemplatesApi.spec.js
deleted file mode 100644
index 2bcd25d153fa6351e4b0d379414aa700c3237f87..0000000000000000000000000000000000000000
diff --git a/test/api/VerificationsApi.spec.js b/test/api/VerificationsApi.spec.js
deleted file mode 100644
index f527c77120cf2bb47c51b568132fc9a1c4fcfcc7..0000000000000000000000000000000000000000
diff --git a/test/model/AccessLevel.spec.js b/test/model/AccessLevel.spec.js
deleted file mode 100644
index df333ac6b8d621756e3e4b1003500200a6e098a0..0000000000000000000000000000000000000000
diff --git a/test/model/AccountStatusEnum.spec.js b/test/model/AccountStatusEnum.spec.js
deleted file mode 100644
index 9494eccd84d64d4990571ce59f343854bd9ae0fc..0000000000000000000000000000000000000000
diff --git a/test/model/ApiKey.spec.js b/test/model/ApiKey.spec.js
deleted file mode 100644
index 1162af3fe588aef0f1922726bf6c69fb5ba89530..0000000000000000000000000000000000000000
diff --git a/test/model/ApiKeyPayload.spec.js b/test/model/ApiKeyPayload.spec.js
deleted file mode 100644
index 78db9127a15bbd3594fc97260ddcfbcb2e876b40..0000000000000000000000000000000000000000
diff --git a/test/model/BodyContentType.spec.js b/test/model/BodyContentType.spec.js
deleted file mode 100644
index c87133563f55974f64f31f5609c0e024bcd377fd..0000000000000000000000000000000000000000
diff --git a/test/model/BodyPart.spec.js b/test/model/BodyPart.spec.js
deleted file mode 100644
index 679120a317c5ec225df5b3c253c004ac79154858..0000000000000000000000000000000000000000
diff --git a/test/model/Campaign.spec.js b/test/model/Campaign.spec.js
deleted file mode 100644
index 5f04726539b7907a4d2c3c2d8d3bd9cd59f62a2f..0000000000000000000000000000000000000000
diff --git a/test/model/CampaignOptions.spec.js b/test/model/CampaignOptions.spec.js
deleted file mode 100644
index d6b9457f49e1e9ab7bd389c3e91219e9d1282979..0000000000000000000000000000000000000000
diff --git a/test/model/CampaignRecipient.spec.js b/test/model/CampaignRecipient.spec.js
deleted file mode 100644
index d0fee3a9caa8db147754253a188aea68668c76d0..0000000000000000000000000000000000000000
diff --git a/test/model/CampaignStatus.spec.js b/test/model/CampaignStatus.spec.js
deleted file mode 100644
index 3beb933b37e04cd3a8578aac4a46898b23dc229d..0000000000000000000000000000000000000000
diff --git a/test/model/CampaignTemplate.spec.js b/test/model/CampaignTemplate.spec.js
deleted file mode 100644
index eea9eb564278ae0357fed8db8d8120c0a97dc7ff..0000000000000000000000000000000000000000
diff --git a/test/model/ChannelLogStatusSummary.spec.js b/test/model/ChannelLogStatusSummary.spec.js
deleted file mode 100644
index fef684c6e0f3acd73c930ab749214e7afe35c913..0000000000000000000000000000000000000000
diff --git a/test/model/CompressionFormat.spec.js b/test/model/CompressionFormat.spec.js
deleted file mode 100644
index 3f1dcc53686054ec90a34f1a5a6d43f5930ab560..0000000000000000000000000000000000000000
diff --git a/test/model/ConsentData.spec.js b/test/model/ConsentData.spec.js
deleted file mode 100644
index 42c84fc56cd36ef41ba537ff1fca896992707b30..0000000000000000000000000000000000000000
diff --git a/test/model/ConsentTracking.spec.js b/test/model/ConsentTracking.spec.js
deleted file mode 100644
index c83b104b44b5f275538df96de82d0f5bdf88bad2..0000000000000000000000000000000000000000
diff --git a/test/model/Contact.spec.js b/test/model/Contact.spec.js
deleted file mode 100644
index b45c78e4986fa8500559676d56a2076977bfa733..0000000000000000000000000000000000000000
diff --git a/test/model/ContactActivity.spec.js b/test/model/ContactActivity.spec.js
deleted file mode 100644
index 167601ce6f6677b653531d961836125325a81bfa..0000000000000000000000000000000000000000
diff --git a/test/model/ContactPayload.spec.js b/test/model/ContactPayload.spec.js
deleted file mode 100644
index a02a5974b0b0a3be96ffb6972ae850a46eb0465a..0000000000000000000000000000000000000000
diff --git a/test/model/ContactSource.spec.js b/test/model/ContactSource.spec.js
deleted file mode 100644
index b15290ce4bc9dd2ccb4d4c46ff506a63bcb2fdbc..0000000000000000000000000000000000000000
diff --git a/test/model/ContactStatus.spec.js b/test/model/ContactStatus.spec.js
deleted file mode 100644
index 5209386b2dccfafc83a470fdc71e8a549540bfe9..0000000000000000000000000000000000000000
diff --git a/test/model/ContactUpdatePayload.spec.js b/test/model/ContactUpdatePayload.spec.js
deleted file mode 100644
index c85a9b01892dee7686c13d62e418bb03aef67773..0000000000000000000000000000000000000000
diff --git a/test/model/ContactsList.spec.js b/test/model/ContactsList.spec.js
deleted file mode 100644
index ec748f8a47d4361a695d1a56ec0e09adc2bb1e72..0000000000000000000000000000000000000000
diff --git a/test/model/DeliveryOptimizationType.spec.js b/test/model/DeliveryOptimizationType.spec.js
deleted file mode 100644
index 072a757d158d7c94ecb086cfaa2e9274962def7c..0000000000000000000000000000000000000000
diff --git a/test/model/EmailContent.spec.js b/test/model/EmailContent.spec.js
deleted file mode 100644
index 8bfa3a90ab1351147907363e347d12faf657ad84..0000000000000000000000000000000000000000
diff --git a/test/model/EmailData.spec.js b/test/model/EmailData.spec.js
deleted file mode 100644
index be7c37a2e8c0662314b7e878d86d6dfff8b53bbc..0000000000000000000000000000000000000000
diff --git a/test/model/EmailJobFailedStatus.spec.js b/test/model/EmailJobFailedStatus.spec.js
deleted file mode 100644
index 1ebc0278be53bbc3ef7e33cc13d2e894b6d38fe1..0000000000000000000000000000000000000000
diff --git a/test/model/EmailJobStatus.spec.js b/test/model/EmailJobStatus.spec.js
deleted file mode 100644
index 078ac2fb4d6a249e3391dd1f527ed40f678faf6b..0000000000000000000000000000000000000000
diff --git a/test/model/EmailMessageData.spec.js b/test/model/EmailMessageData.spec.js
deleted file mode 100644
index e62f49ba581b9f7fb7ea900fe1368bda4e3f5f12..0000000000000000000000000000000000000000
diff --git a/test/model/EmailPredictedValidationStatus.spec.js b/test/model/EmailPredictedValidationStatus.spec.js
deleted file mode 100644
index 5ef8e1887fe5ee0c75ed3847f3fd50fa296d4413..0000000000000000000000000000000000000000
diff --git a/test/model/EmailRecipient.spec.js b/test/model/EmailRecipient.spec.js
deleted file mode 100644
index a1a8f3a5f36bcd901decffaef6c8fb81ba72c511..0000000000000000000000000000000000000000
diff --git a/test/model/EmailSend.spec.js b/test/model/EmailSend.spec.js
deleted file mode 100644
index f16d11dc1888f9b443af57109e4135fc1474c381..0000000000000000000000000000000000000000
diff --git a/test/model/EmailStatus.spec.js b/test/model/EmailStatus.spec.js
deleted file mode 100644
index 25993d13ca18ab005c63a316603048222d16d50b..0000000000000000000000000000000000000000
diff --git a/test/model/EmailTransactionalMessageData.spec.js b/test/model/EmailTransactionalMessageData.spec.js
deleted file mode 100644
index c859bc17c5466fb137561fae0c70720e3c885777..0000000000000000000000000000000000000000
diff --git a/test/model/EmailValidationResult.spec.js b/test/model/EmailValidationResult.spec.js
deleted file mode 100644
index 62856b8b3e422c34d013d594754f08f534102fe6..0000000000000000000000000000000000000000
diff --git a/test/model/EmailValidationStatus.spec.js b/test/model/EmailValidationStatus.spec.js
deleted file mode 100644
index 66fa1542a90568a2215f4223871bbefd170b749c..0000000000000000000000000000000000000000
diff --git a/test/model/EmailView.spec.js b/test/model/EmailView.spec.js
deleted file mode 100644
index 0e38bb06cddb05d9012e2a9f2b6ce5ef7c05b903..0000000000000000000000000000000000000000
diff --git a/test/model/EmailsPayload.spec.js b/test/model/EmailsPayload.spec.js
deleted file mode 100644
index b4f8960a6727b50fb92fc8a901a94a287b48f34a..0000000000000000000000000000000000000000
diff --git a/test/model/EncodingType.spec.js b/test/model/EncodingType.spec.js
deleted file mode 100644
index b3290f187fe12b714308649fef08d4f809949718..0000000000000000000000000000000000000000
diff --git a/test/model/EventType.spec.js b/test/model/EventType.spec.js
deleted file mode 100644
index 6d033ed5fa1bfc652b8a16741b882c9a2f74ce02..0000000000000000000000000000000000000000
diff --git a/test/model/EventsOrderBy.spec.js b/test/model/EventsOrderBy.spec.js
deleted file mode 100644
index 2b2b5f2c3b60ec4f6e32dbf7192997c6b7ca9588..0000000000000000000000000000000000000000
diff --git a/test/model/ExportFileFormats.spec.js b/test/model/ExportFileFormats.spec.js
deleted file mode 100644
index 6fce4b8f846d18de5fdc39b54e0fa4e7adcb7651..0000000000000000000000000000000000000000
diff --git a/test/model/ExportLink.spec.js b/test/model/ExportLink.spec.js
deleted file mode 100644
index f9e0343cf851d83b622af9ca72cd6deff9849e4f..0000000000000000000000000000000000000000
diff --git a/test/model/ExportStatus.spec.js b/test/model/ExportStatus.spec.js
deleted file mode 100644
index 683b7e5a00dd4ec76f8bdb41a1402c9c0d49dcf5..0000000000000000000000000000000000000000
diff --git a/test/model/FileInfo.spec.js b/test/model/FileInfo.spec.js
deleted file mode 100644
index 46508c06bfbddbd88b26c9432e8c540ed1e96854..0000000000000000000000000000000000000000
diff --git a/test/model/FilePayload.spec.js b/test/model/FilePayload.spec.js
deleted file mode 100644
index 0f9144a1ff4d86a65a16c73d8a2c3ec63bfd8052..0000000000000000000000000000000000000000
diff --git a/test/model/FileUploadResult.spec.js b/test/model/FileUploadResult.spec.js
deleted file mode 100644
index e811c9f38277f5fabf6afb1483389dff5211264c..0000000000000000000000000000000000000000
diff --git a/test/model/InboundPayload.spec.js b/test/model/InboundPayload.spec.js
deleted file mode 100644
index 6448a89edada3dbdd998f2f3582f839faed038d3..0000000000000000000000000000000000000000
diff --git a/test/model/InboundRoute.spec.js b/test/model/InboundRoute.spec.js
deleted file mode 100644
index c7766fb82aea1cd0dd734fd7535513419844c7ad..0000000000000000000000000000000000000000
diff --git a/test/model/InboundRouteActionType.spec.js b/test/model/InboundRouteActionType.spec.js
deleted file mode 100644
index 1c3ac837677f7b2cddf6fa40533d416d3a3a5a7b..0000000000000000000000000000000000000000
diff --git a/test/model/InboundRouteFilterType.spec.js b/test/model/InboundRouteFilterType.spec.js
deleted file mode 100644
index a8780f693b01adebd064f5c34fa98566e3ffd1f1..0000000000000000000000000000000000000000
diff --git a/test/model/ListPayload.spec.js b/test/model/ListPayload.spec.js
deleted file mode 100644
index 227d8dd0b5596f58ced78d4dcd958adbfb990545..0000000000000000000000000000000000000000
diff --git a/test/model/ListUpdatePayload.spec.js b/test/model/ListUpdatePayload.spec.js
deleted file mode 100644
index fa20e6b652f57feb796402d20fe518d5bb64ffb5..0000000000000000000000000000000000000000
diff --git a/test/model/LogJobStatus.spec.js b/test/model/LogJobStatus.spec.js
deleted file mode 100644
index 3cec237e808903304fc9ec1f72839fcc2b790714..0000000000000000000000000000000000000000
diff --git a/test/model/LogStatusSummary.spec.js b/test/model/LogStatusSummary.spec.js
deleted file mode 100644
index 3230312f8812abdc4b11dad4de5f54d4243457d6..0000000000000000000000000000000000000000
diff --git a/test/model/MergeEmailPayload.spec.js b/test/model/MergeEmailPayload.spec.js
deleted file mode 100644
index 12d59365ef63385795baa6fabee2580468a46706..0000000000000000000000000000000000000000
diff --git a/test/model/MessageAttachment.spec.js b/test/model/MessageAttachment.spec.js
deleted file mode 100644
index 78db3faa899d13b205330c9534fc8cc2e2223166..0000000000000000000000000000000000000000
diff --git a/test/model/MessageCategory.spec.js b/test/model/MessageCategory.spec.js
deleted file mode 100644
index 2b1be6a50e50387a91d952bb7ece09ba4f278e61..0000000000000000000000000000000000000000
diff --git a/test/model/NewApiKey.spec.js b/test/model/NewApiKey.spec.js
deleted file mode 100644
index bbc3dfc5834249a297dc4e176ecd8e95f23fc32f..0000000000000000000000000000000000000000
diff --git a/test/model/NewSmtpCredentials.spec.js b/test/model/NewSmtpCredentials.spec.js
deleted file mode 100644
index 6822efdd5829c623f379db0a9632b801646e1501..0000000000000000000000000000000000000000
diff --git a/test/model/Options.spec.js b/test/model/Options.spec.js
deleted file mode 100644
index ecec793396f1e7c39a381b2b4a3eff18659826f3..0000000000000000000000000000000000000000
diff --git a/test/model/RecipientEvent.spec.js b/test/model/RecipientEvent.spec.js
deleted file mode 100644
index b8634eedab9c013c508e75ab47758c5d886c5862..0000000000000000000000000000000000000000
diff --git a/test/model/Segment.spec.js b/test/model/Segment.spec.js
deleted file mode 100644
index 9f4fa8de07b45d5ad8ad046a7df6d421fec4d4fe..0000000000000000000000000000000000000000
diff --git a/test/model/SegmentPayload.spec.js b/test/model/SegmentPayload.spec.js
deleted file mode 100644
index 666eda23bf9313c3e64a24881ed6921071a219c9..0000000000000000000000000000000000000000
diff --git a/test/model/SmtpCredentials.spec.js b/test/model/SmtpCredentials.spec.js
deleted file mode 100644
index 5c3e92130688b401f13a7d8d8cda6ceea7d65fab..0000000000000000000000000000000000000000
diff --git a/test/model/SmtpCredentialsPayload.spec.js b/test/model/SmtpCredentialsPayload.spec.js
deleted file mode 100644
index 1c961e1c9993ec32c6fa92681a3ced9ec7e98b76..0000000000000000000000000000000000000000
diff --git a/test/model/SortOrderItem.spec.js b/test/model/SortOrderItem.spec.js
deleted file mode 100644
index 313677f39f0c185cbb3cfb06049fdde5a71c308f..0000000000000000000000000000000000000000
diff --git a/test/model/SplitOptimizationType.spec.js b/test/model/SplitOptimizationType.spec.js
deleted file mode 100644
index 57e0d8ad2ee8de7c36872b074d8059c47dc414bc..0000000000000000000000000000000000000000
diff --git a/test/model/SplitOptions.spec.js b/test/model/SplitOptions.spec.js
deleted file mode 100644
index fbab4341fc92aea4c3fc1ebc86e6bbe808c19cdf..0000000000000000000000000000000000000000
diff --git a/test/model/SubAccountInfo.spec.js b/test/model/SubAccountInfo.spec.js
deleted file mode 100644
index d1abf5b757b2a17369cf6f0fc93d5b336217c8e0..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountEmailCreditsPayload.spec.js b/test/model/SubaccountEmailCreditsPayload.spec.js
deleted file mode 100644
index 3494548f1960f5db3bad6907f42678e07ee2fa61..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountEmailSettings.spec.js b/test/model/SubaccountEmailSettings.spec.js
deleted file mode 100644
index a14e4033f77d3dc9c7ecc8caa2d0a7872bbefeb5..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountEmailSettingsPayload.spec.js b/test/model/SubaccountEmailSettingsPayload.spec.js
deleted file mode 100644
index 411e43841120dad2e82df30f736c7c898143460a..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountPayload.spec.js b/test/model/SubaccountPayload.spec.js
deleted file mode 100644
index 0ae15eb18241cab21232a90f15180da793767c02..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountSettingsInfo.spec.js b/test/model/SubaccountSettingsInfo.spec.js
deleted file mode 100644
index 5f8c4bca40d38260340fff680d689ce004056630..0000000000000000000000000000000000000000
diff --git a/test/model/SubaccountSettingsInfoPayload.spec.js b/test/model/SubaccountSettingsInfoPayload.spec.js
deleted file mode 100644
index 0771f16432d967b63abe0162663de61a8a25595b..0000000000000000000000000000000000000000
diff --git a/test/model/Suppression.spec.js b/test/model/Suppression.spec.js
deleted file mode 100644
index 5c2e3f0679670132428551c4fc613798c48ec114..0000000000000000000000000000000000000000
diff --git a/test/model/Template.spec.js b/test/model/Template.spec.js
deleted file mode 100644
index 6c3749094c982912654e41f8ecd5530185abb116..0000000000000000000000000000000000000000
diff --git a/test/model/TemplatePayload.spec.js b/test/model/TemplatePayload.spec.js
deleted file mode 100644
index ea74d52796c98673878f104ed8456df93a966e13..0000000000000000000000000000000000000000
diff --git a/test/model/TemplateScope.spec.js b/test/model/TemplateScope.spec.js
deleted file mode 100644
index 29aa364d4a6076387f34a9c1385c4ea34fa9ea55..0000000000000000000000000000000000000000
diff --git a/test/model/TemplateType.spec.js b/test/model/TemplateType.spec.js
deleted file mode 100644
index 672367894cf0d27ad8a4b6a4c843f07624d3f48f..0000000000000000000000000000000000000000
diff --git a/test/model/TransactionalRecipient.spec.js b/test/model/TransactionalRecipient.spec.js
deleted file mode 100644
index e2c4984463646674c4f3ce078ba096a6b89911d0..0000000000000000000000000000000000000000
diff --git a/test/model/Utm.spec.js b/test/model/Utm.spec.js
deleted file mode 100644
index 75486c8f14236206227eb8e7af81b470c636e67b..0000000000000000000000000000000000000000
diff --git a/test/model/VerificationFileResult.spec.js b/test/model/VerificationFileResult.spec.js
deleted file mode 100644
index f983e51f46fd26cd5a90fdb3d3bd9fd485da5622..0000000000000000000000000000000000000000
diff --git a/test/model/VerificationFileResultDetails.spec.js b/test/model/VerificationFileResultDetails.spec.js
deleted file mode 100644
index a9b8f2a31da47d58853fa3abbb7f91befecf6119..0000000000000000000000000000000000000000
diff --git a/test/model/VerificationStatus.spec.js b/test/model/VerificationStatus.spec.js
deleted file mode 100644
index 5a2521728dbcfcae83469c551e0b0975defcb4a7..0000000000000000000000000000000000000000
